example_project_bzl_4/third_party/ncurses/ncurses.nix

```diff
@@ -0,0 +1,85 @@
+let pkgs = import <nixpkgs> {};
```

Comment:
See comment regarding `readlines`  - perhaps the code duplication is not needed

Reply:
Done, see below. I had to abandon Unicode for now, as it ended up with infinite symlink recursion (the link fixup script is a bit flaky).